plugin/evm/vm_test.go

```diff
@@ -92,97 +101,158 @@ var (
 		return string(b)
 	}
 
-	genesisJSONPreSubnetEVM = genesisJSON(params.TestPreSubnetEVMChainConfig)
-	genesisJSONSubnetEVM    = genesisJSON(params.TestSubnetEVMChainConfig)
-	genesisJSONDurango      = genesisJSON(params.TestDurangoChainConfig)
-	genesisJSONEtna         = genesisJSON(params.TestEtnaChainConfig)
-	// genesisJSONFortuna      = genesisJSON(params.TestFortunaChainConfig)
-	genesisJSONGranite = genesisJSON(params.TestGraniteChainConfig)
-	genesisJSONLatest  = genesisJSONGranite
+	// These will be initialized after init() runs
+	// forkToChainConfig maps a fork to a chain config
+	forkToChainConfig = map[upgradetest.Fork]*params.ChainConfig{
+		upgradetest.Durango: params.TestDurangoChainConfig,
+		upgradetest.Etna:    params.TestEtnaChainConfig,
+		// upgradetest.Fortuna: params.TestFortunaChainConfig,
+		upgradetest.Granite: params.TestGraniteChainConfig,
+	}
+
+	genesisJSONPreSubnetEVM string
```

Comment:
can we initialize them here as `genesisJSONPreSubnetEVM = toGenesisJSON(params.TestPreSubnetEVMChainConfig)`